tools/assignmentPlacer.py

```diff
@@ -0,0 +1,28 @@
+import os, sys
```

Comment:
can we rename this to `assignment_placer.py` to better follow how other files are named.